README.md

```diff
@@ -0,0 +1,198 @@
+
+    To test a rhel7-based ruby-2.0 image, you need to run the test on a properly
+    subscribed RHEL machine.
+
+    ```
+    $ cd sti-ruby
+    $ make test TARGET=rhel7 VERSION=2.0
+    ```
+
+*  **CentOS based image**
+
+    ```
+    $ cd sti-ruby
+    $ make test VERSION=2.0
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of Ruby. Since we are now supporting only version `2.0`
+you can omit this parameter.**
+
+
+Repository organization
+------------------------
+* **`<ruby-version>`**
+
+    * **Dockerfile**
+
+        CentOS based Dockerfile.
+
+    * **Dockerfile.rhel7**
+
+        RHEL based Dockerfile. In order to perform build or test actions on this
+        Dockerfile you need to run the action on properly subscribed RHEL machine.
+
+    * **`.sti/bin/`**
+
+        This folder contains scripts that are run by [STI](https://github.com/openshift/source-to-image):
+
+        *   **assemble**
+
+            Is used to install the sources into location from where the application
+            will be run and prepare the application for deployment (eg. installing
+            modules using npm, etc..)
+
+            Following [markers](#markers) take effect in `assamble` script:
+            - disable_asset_compilation
+
+        *   **run**
+
+            This script is responsible for running the application, by using the
+            application web server.
+
+    * **`ruby/`**
+
+        This folder contains file with commonly used modules.
+
+    * **`test/`**
+
+        This folder is containing [STI](https://github.com/openshift/source-to-image)
+        test framework with simple Rack server.
+
+        * **`test-app/`**
+
+            Simple Rack server used for testing purposes in the [STI](https://github.com/openshift/source-to-image) test framework.
+
+        * **run**
+
+            Script that runs the [STI](https://github.com/openshift/source-to-image) test framework.
+
+* **`hack/`**
+
+    Folder contains scripts which are responsible for build and test actions performed by the `Makefile`.
+
+Image name structure
+------------------------
+##### Structure: openshift/1-2-3
+
+1. Platform name - ruby
+2. Platform version(without dots)
+3. Base builder image - centos7/rhel7
+
+Examples: `openshift/ruby-20-centos7`, `openshift/ruby-20-rhel7`
+
+
+Markers
+---------------------
+Markers are empty files named after their expected behavior upon running
+[STI](https://github.com/openshift/source-to-image) scripts. Markers
+should be placed in a subdirectory `.sti/markers/` at the root of your
+application folder.
+
+*  **disable_asset_compilation**
+
+    Takes effect in the `assamble` script. Will prevent assets to be compiled upon
+    application deployment. This marker should be used when deploying application
+    with assets which are already compiled.
+
+
+Environment variables
+---------------------
+
```

Comment:
They don't influence the assets compilation (AFAIK), but they influence the runtime (with development you get more verbose logs, error pages, etc...). Also we should pick up just one as Rails can detect RACK_ENV itself.